src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -597,10 +597,15 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
                         resolve(serializeData(this.store.downloadData));
                     }
                     else {
-                        reject();
+                        // even if loading of all lazy data fails, still allow downloading of the partial data
+                        reject(serializeData(this.store.downloadData));
```

Comment:
this is a bit nonidiomatic... Id say instead you should resolve either way with `{ status: "complete"|"incomplete", data: serializeData(this.store.downloadData) }`

Reply:
sounds good

Reply:
added a new type in `CopyDownloadControls`:

```
interface ICopyDownloadData {
    status: 'complete'|'incomplete';
    text: string;
}
```